flink-connector-opensearch/src/main/java/org/apache/flink/streaming/connectors/opensearch/util/IgnoringFailureHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.connectors.opensearch.util;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.streaming.connectors.opensearch.ActionRequestFailureHandler;
+import org.apache.flink.streaming.connectors.opensearch.RequestIndexer;
+
+import org.opensearch.action.ActionRequest;
+
+/** Ignores all kinds of failures and drops the affected {@link ActionRequest}. */
+@Internal
+public class IgnoringFailureHandler implements ActionRequestFailureHandler {
```

Comment:
I believe these 2 handlers (IgnoringFailureHandler and RetryRejectedExecutionFailureHandler) are provided as an alternatives for configuring OpensearchSink::setFailureHandler

Reply:
Yes and no. A user _could_ set the failure handler, but this implementation here is marked as `@Internal`; presumably because they were targeting the Table API connector.
See `ElasticsearchConfiguration#getFailureHandler`

_If_ we want to be be a drop-in replacement for the ES table api connector, then we'd need to pull in all of FLINK-26638, which also includes going back to the old ES sink for the Table API.

It's a bit of an unsolved question on the ES connector side as to what will happen with these failure handlers :/

Reply:
Sure, thanks, I will drop them